### ISSUE TITLE
Accessibility: with Orca (ctrl+e) does not work

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/vscode#186434]
Testing #186217

![image](https://github.com/microsoft/vscode/assets/3840081/ef326e28-73a1-4afa-994f-c7b755049b61)

Shows Ctrl + e , but that is mapped to "Go to file"

**Comment by meganrogge** [comment]
your image didn't work

**Comment by karthiknadig** [comment]
Updated the image.

**Comment by alexr00** [comment]
Verified with Windows.

**Comment by karthiknadig** [comment]
Verified that this is now working with Linux and Orca
